pcsx2-qt/MainWindow.cpp

```diff
@@ -141,6 +142,8 @@ void MainWindow::initialize()
 #endif
 	m_ui.setupUi(this);
 	setupAdditionalUi();
+	MouseWheelFilter* wf = new MouseWheelFilter(this);
```

Comment:
Could you move this to the end of `setupAdditionalUi()`? Looks a bit out of place here.